sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/ExecutorResourceMapper.java

```diff
@@ -14,8 +16,9 @@
  * Handles cross-referencing a preexisting {@link Protos.ExecutorInfo}'s current {@link Protos.Resource}s against a set
  * of expected {@link VolumeSpec}s for that Executor.
  */
-public class ExecutorResourceMapper extends AbstractResourceMapper {
+public class ExecutorResourceMapper {
 
+    final Logger logger;
```

Comment:
This could be `private`, but this can be fixed in a separate PR if no other changes are needed